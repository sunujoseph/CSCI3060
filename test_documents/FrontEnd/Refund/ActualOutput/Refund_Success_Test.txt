Enter Username: 
Enter Buyer Username: 
Enter Seller Username: 
Enter Minimum Bid: 
