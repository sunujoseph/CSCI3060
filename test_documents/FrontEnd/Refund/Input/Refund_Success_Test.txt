login
admin1
refund
item2
userFS
userSS
990990
