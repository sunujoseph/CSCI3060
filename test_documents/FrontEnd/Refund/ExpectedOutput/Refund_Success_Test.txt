Please log in to the system: 
Please enter your username: 
Enter command: 
Enter Item Name: 
Enter buyer username: 
Enter seller username: 
Enter amount of credit: 
Your request for refund for your Sony PlayStation 4 from ken_adams56 for $5000 has been sent...
ERROR: Seller ken_adams56 amount of credit did not decrease the right amount. Amount decreased is 256.
ERROR: Buyer john123smith amount of credit did not increase the right amount. Amount increased is 256. 
ERROR: Full Refund cannot be done at this time.
Goodbye!